harper-typst/src/typst_translator.rs

```diff
@@ -203,6 +203,41 @@ impl<'a> TypstTranslator<'a> {
             )
         };
 
+        // Parse a function call
+        let parse_func_call = |func: FuncCall| {
+            let parse_args_without = |pos: bool, nameds: &[&str]| {
```

Comment:
Can you clarify what `pos` represents?

It looks like every time you call this you pass true in for pos.